src/domUpdates.js

```diff
@@ -173,8 +174,20 @@ const closeRecipe = () => {
   clickedRecipe.classList.add("hidden");
   clickedRecipe.classList.remove("flex");
   clickedRecipe.classList.remove("fade-in");
+  ingredientsList.innerHTML = '';
 };
 
+const getIngredients = currentRecipeCard => {
+  createIngredientsHTML(currentRecipeCard.ingredients);
+};
+
+const createIngredientsHTML = ingredients => {
+  ingredients.forEach(ingredient => {
+    ingredientsList.innerHTML += `
+      <label><input type="checkbox" class="ingredient-text" value="value">${ingredient.amount} ${ingredient.unit} ${ingredient.name}</label>
```

Comment:
How would you feel about making separate label and input tags and adding `id` to input and `for` to label to connect the two? From my understanding, I think that screen readers rely on that type of connection to know which input a certain label is for. 

Let me know if that is not correct, but I think that it the standard way to go about using the two in conjunction.

Reply:
yep can def make that change - didn't that was the standard for accessibility!